packages/runtime/container-runtime/src/opTelemetry.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IDeltaManager } from "@fluidframework/container-definitions";
+import {
+    IDocumentMessage,
+    ISequencedDocumentMessage,
+} from "@fluidframework/protocol-definitions";
+import { isSystemMessage } from "@fluidframework/protocol-base";
+import { utf8ByteLength } from "@fluidframework/runtime-utils";
+
+export class OpTracker {
+    private _nonSystemOpCount: number = 0;
+    public get nonSystemOpCount(): number {
+        return this._nonSystemOpCount;
+    }
+
+    private _opsSizeAccumulator: number = 0;
+    public get opsSizeAccumulator(): number {
+        return this._opsSizeAccumulator;
+    }
+
+    public constructor(
+        deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
+        disabled: boolean,
+    ) {
+        if (disabled) {
+            return;
+        }
+
+        deltaManager.on("op", (message) => {
+            this._nonSystemOpCount += isSystemMessage(message) ? 0 : 1;
+            const stringContents = typeof message.contents === "string" ?
+                message.contents :
+                JSON.stringify(message.contents);
+            this._opsSizeAccumulator += utf8ByteLength(stringContents);
```

Comment:
I'm not sure we need that kind of precision (I actually have same though about summarization code :)).
I think character length is more than enough.
I remember I provided this feedback back in the day when summarization code was added (we were doing conversion to utf8 buffer to calculate lenght) and developer (I do not remember who it was, probably Wes) implement utf8ByteLength() and did some measurements, and it was Ok'ish. (maybe we can find that PR / issue).
Here, though, calculation is on critical (responsiveness) path, so requirements are a bit higher, and I'm not sure we want to pay the extra cost, even if it's rather small.

What do you think?

Reply:
> I'm not sure we need that kind of precision (I actually have same though about summarization code :)).

:) That's exactly what I was going for. For some consistency between the summarizer and this wrt to the size of stuff. I agree we're probably fine with just the character size.